.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,11 +1,18 @@
-#### Description
-<!--- A brief description of your changes. --->
+### Description
+ ...A brief description of your changes.
+ ...Make sure your pull-request title is clearly named
 
-#### Changes
-<!--- Summary of changes to guide the reviewer(s). --->
+### Changes
+...Summary of changes to guide the reviewer(s)
+...Whats been tested, manually or automatically
+...Has the documentation been updated, where relevant
```

Comment:
should you write about documentation in changes or additional notes

Reply:
It means ' have you updated the documentation in the codebase', maybe this could be better worded? Suggestions?